Data/Interfaces/IHotelRoom.cs

```diff
@@ -9,14 +9,14 @@ namespace AsyncInn.Data.Interfaces
     public interface IHotelRoom
 
     {
-        Task<IEnumerable<Hotel>> GetHotelRooms();
+        Task<IEnumerable<HotelModel>> GetHotelRooms(int hotelId);
 
-        Task<Hotel> GetHotelRoom(int id);
+        Task<HotelModel> GetHotelRoom(int hotelId, int roomNumber);
 
-        Task<bool> DeleteHotelRoom(int id);
+        Task<bool> DeleteHotelRoom(int hotelId, int roomNumber);
 
-        Task PostHotelRoom(Hotel hotel);
+        Task PostHotelRoom(int hotelId, int roomNumber);
```

Comment:
You probably want some sort of `CreateHotelRoom` model with `RoomNumber`, `PetFriendly`, etc.